library/ns1_zone.py

```diff
@@ -110,173 +140,323 @@
 
 author:
   - 'Matthew Burtless (@mburtless)'
```

Comment:
Not to rain on your parade but should this also be changed to NS1?

Reply:
omg yes, good catch. Hold over from when we were trying to get this merged to core and had follow their module dev guide.

Reply:
I'm actually not sure if `ansible-doc` will continue to work on this module if we don't match the format prescribed in https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-block.  Will test.

Reply:
Looks like `ansible-doc` DGAF, but turned out I needed to remove the `doc_fragment` ref as well.  Both have been updated.